cloud-functions/cf-dsm-file-scanner/dsm-gcp-setup/dsm-setup.sh

```diff
@@ -36,20 +38,13 @@ echo "creating the $trigger_topic_name pubsub topic that triggers the FileScanne
 ./create_topic_and_sub.sh $project_name $trigger_topic_name ""
 
 
-scan_result_topic_name="dsm-file-antivirus-result"
-scan_result_subscription_name="dsm-file-antivirus-result-sub"
-
-# Call the create_topic_and_sub.sh script to create the pubsub topic that has the scanning result
-echo "creating the $scan_result_topic_name pubsub topic that triggers the FileScanner"
-./create_topic_and_sub.sh $project_name $scan_result_topic_name $scan_result_subscription_name
```

Comment:
This was not used anywhere but in the cf-deploy.sh 

The naming comes from old cloud function